stage-process.md

```diff
@@ -0,0 +1,115 @@
+# CG/WG Proposal Stages
+
+Author: @bumblefudge, @samuelgoto
+Created: 09/26/2024
+Last update: 09/26/2024
+Status: proposed CG charter amendment
+
+This is a proposal to break proposals into 5 stages of maturity, with clear guidelines and requirements to advance them:
+
+* [Stage 0](#stage0): **Exploration** of a Problem Space and/or user stories
+* [Stage 1](#stage1): **Incubation** of targeted candidate solutions
+* [Stage 2](#stage2): **Formalization** a preferred Proposal
+* [Stage 3](#stage3): **Implementation** of the preferred Proposal
+* [Stage 4](#stage4): **Publication** of a Proposed Recommendation
+
+These stages support the W3C process.
+If there is ever any question between W3C process requirements and how the groups will progress their work, the W3C process has precedent.
+
+## Stage 0: Exploration
+
+The purpose of Stage 0 proposals is to allow anyone to raise and explore the Problem Space without asking for permission. 
+
+  * What's needed from individuals?
+    * [ ] Typically, a detailed (but IP-safe, high-level) issue describing a user story or problem
```

Comment:
Oh, I love the "IP-safe" delineation ... would love to take that and merge that into the FedID one ...